src/config/axios.js

```diff
@@ -40,6 +45,27 @@ axiosConfig.interceptors.response.use(
   },
 );
 
-axiosConfig.interceptors.response.use((response) => response?.data);
+axios.interceptors.response.use((response) => response?.data);
 
-export {axiosConfig};
+export const logAxiosError = (error) => {
```

Comment:
Try to declare `logAxiosError` before using it line 34.

Reply:
Done !